File: class1ex.java
public class class1ex {

    public String hello_comp(String B) {
        return B;
    }

    public String greet(String C) {
        return C;
    }

    public static void main(String[] args) {
        class1ex greeting = new class1ex();
        String x = greeting.greet("Hello Calvin");
        String y = greeting.greet("I am 21 years old");
        System.out.println(x);
        System.out.println(y);
    }
}

File: class1ex.py
class Greeting:
    def __init__(self ,x ,y):
        self.x = x
        self.y = y

    def hello_comp(self):
        return "Hello Calvin"

    def greet(self):
        return "my age 21"

x = Greeting("Greeting1", "greeting2")
print(x.hello_comp())
print(x.greet())
